### PR TITLE
Info Pages Admin UI

### PR DESCRIPTION
**Pull request by MatthiasReumann**
### Motivation and Context
Admin Info Page UI was outdated and bugged.

### Description
Update UI with `.form-container`s. See Screenshots.

_Info: Using button classes from #792_

### Steps for Testing

Prerequisites:
- 1 Admin
- Admin Edit-Info-Pages page

1. Log in
2. Head to "Info pages"

### Screenshots
<details>
  <summary>Before:</summary>
image
</details>

<details>
  <summary>After:</summary>
image

I changed the title to `Info Pages`. 🫠
</details>

### CODE REVIEW

File: web/template/admin/admin_tabs/info-pages.gohtml [hunk 1]
```diff
@@ -1,54 +1,95 @@
...
+                        <button id="save-{{$i}}"
+                                type="submit"
+                                class="btn h-fit"
+                                :disabled="newContent === oldContent">
+                            Save
+                        </button>
+                    </div>
+                </form>
+
+                <!--
```
Comment (alexanderstephan) [inline web/template/admin/admin_tabs/info-pages.gohtml]: Can't we remove the code that is commented out?

**Review by alexanderstephan — CHANGES_REQUESTED** [review]
Looks really nice. 🎉  Just one small question.

File: web/template/admin/admin_tabs/info-pages.gohtml (hunk 1 above)
Comment (MatthiasReumann) [reply web/template/admin/admin_tabs/info-pages.gohtml]: Oh yeah, definitely. Sorry for that :/

**Review by alexanderstephan — APPROVED** [review]
🎉

**Pull request merged**